app/controllers/application_controller/tenancy.rb

```diff
@@ -9,11 +9,10 @@ module ApplicationController::Tenancy
   end
 
   def current_tenant
-    # current_user.try(:current_group).try(:tenant) || Tenant.default_tenant
     @current_tenant ||=
```

Comment:
I'm not familiar with this code, do we want to cache this here?  what happens if the user changes it's group/tenant?  Is this memoized for the life of the request or only once? What happens if you log out and log back in as a different user?